.gitmodules

```
@@ -0,0 +1,3 @@
+[submodule ".test/data"]
```

I think this needs to read:
```suggestion
[submodule ".test/ngs-test-data"]
```